.gitignore

```diff
@@ -23,3 +23,6 @@
 /config/master.key
 
 /coverage
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
Is there anything in here we will need to have on our local machines?

Reply:
yeah, the username and password of the glow up gmail. I'll post them in slack